### PR TITLE
Adding documentation and constant

### PR DESCRIPTION
**Pull request by jeremyf**
## What type of PR is this? (check all applicable)

- [x] Documentation Update

## Description

This loosely relates to a request for information from Success team
about whether or not viewing a draft article counts towards it's stats.

## Related Tickets & Documents

Related to questions about draft previews counting towards page views.

## QA Instructions, Screenshots, Recordings

None.

### UI accessibility concerns?

None.

## Added/updated tests?

- [x] No

## [Forem core team only] How will this change be communicated?

- [ ] This change does not need to be communicated.

### CODE REVIEW

File: app/assets/javascripts/initializers/initializeBaseTracking.js.erb [hunk 1]
```diff
@@ -111,6 +111,9 @@ function trackCustomImpressions() {
 
     // page view
     if (ArticleElement && tokenMeta && !isBot) {
+      // See https://github.com/forem/forem/blob/05413f101ab92eb7e25cf5b098919e21179be602/app/controllers/page_views_controller.rb
```
Comment (aitchiss) [inline app/assets/javascripts/initializers/initializeBaseTracking.js.erb:114]: This link is going to a particular commit, should we link to the current `main` branch for this file instead?
```suggestion
      // See https://github.com/forem/forem/blob/main/app/controllers/page_views_controller.rb
```

File: app/controllers/page_views_controller.rb [hunk 2]
```diff
@@ -3,12 +3,23 @@ class PageViewsController < ApplicationMetalController
...
   include ActionController::Head
 
+  # Each time we have a non-authenticated visitor, we have a 10% chance that we will record that
+  # specific impression as a page view.  When we do record that specific impression as a page view,
+  # we want to give credit for all likely page views that we didn't record (e.g., the 90% or so).
+  #
+  # @note Yes, this is a very verbose constant name.  Apologies, don't type it.  But I [@jeremyf]
+  #       want it here to explain a magic number.
+  #
+  # @see https://github.com/forem/forem/blob/05413f101ab92eb7e25cf5b098919e21179be602/app/assets/javascripts/initializers/initializeBaseTracking.js.erb#L113-L117
```
Comment (aitchiss) [inline app/controllers/page_views_controller.rb:13]: Same with this one, should we point to `main`? (although I guess the line references could fall out of sync if we do, so maybe best to keep as is 🤔 )
```suggestion
  # @see https://github.com/forem/forem/blob/main/app/assets/javascripts/initializers/initializeBaseTracking.js.erb#L113-L117
```

**Review by aitchiss — APPROVED** [review]
Thanks for making this all much clearer! Left a couple of suggestions but LGTM

File: app/controllers/page_views_controller.rb (hunk 2 above)
Comment (jeremyf) [reply app/controllers/page_views_controller.rb:13]: Yeah, a head scratcher and part of why I wanted it to be a particular SHA.  "main" makes for a shorter URL but I'm always hesitant to consider how something might move and "get lost".

`main` has the "this is the likely ongoing state" which I believe is most true to intention.

**Review by juliannatetreault — APPROVED** [review]

**Pull request merged**